libr/config/config.c

```diff
@@ -55,8 +55,8 @@ static void config_print_value_json(RConfig *cfg, RConfigNode *node) {
 		if (!strncmp (val, "0x", 2)) {
 			ut64 n = r_num_get (NULL, val);
 			cfg->cb_printf ("%"PFMT64d, n);
-		} else if (r_str_isnumber (val) || r_str_is_bool (val)) {
-			cfg->cb_printf ("%s", val);
+		} else if (r_str_isnumber (val) || (*val /* HACK */ && r_str_is_bool (val))) {
```

Comment:
what is this hack? what does it do? it's not clear to me

Reply:
see https://github.com/radareorg/radare2/pull/15859#issuecomment-576605121